Report/sections/3_implementation/interpreter.tex

```diff
@@ -0,0 +1,149 @@
+\section{The Game}
+Following the code generation phase is the final result of the code compilation - the game itself. 
+This is built using the Unity engine and interprets the intermediate models in order to build the world defind by the user in the \dazel{} source code. As can be seen in \figref{Dazel_Title}, the user is presented with a few options when launching the application.
```

Comment:
```suggestion
The game was built using the Unity engine. It interprets the intermediate models in order to build the world defind by the user in the \dazel{} source code. As can be seen in \figref{Dazel_Title}, the user is presented with a few options when launching the application.
```

Reply:
Personal opinion: one sentence, one idea. This is a nit - you can disregard if you don't agree.